### PR TITLE
Added development guide to  readme

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,17 @@ npm install
...
+```bash
+npm install
+npm start -- --debug
+```
+
+2. Set `debug` to `true` in the config. The config file is stored in `~/.config/configstore/Moeditor.json`.
+
+3. `Ctrl` + `Shift` + `I` in Linux / Windows or `Command` + `Option` + `I` in OS X / macOS to toggle devtools for a window.
+
 In China, you may want to replace npm with cnpm for a faster download speed.
```
Comment (Menci) [inline README.md]: This line should be in **Build**.

File: README.md [hunk 2]
```diff
@@ -20,6 +20,17 @@ npm install
 npm start
 ```
 
+# Development
+1. Add `--debug` to the command line args:
+```bash
+npm install
```
Comment (Menci) [inline README.md]: There shouldn't be `npm install` since it's mentioned in **Building**.

File: README.md [hunk 3]
```diff
@@ -20,6 +20,17 @@ npm install
 npm start
 ```
 
+# Development
```
Comment (Menci) [inline README.md]: Tell clearly we can use ANY of below.

File: README.md (hunk 1 above)
Comment (amilajack) [inline README.md]: Which line are you referring to? Github's comment system doesnt make this very clear

Comment (Menci) [inline README.md]: > In China, you may want to replace npm with cnpm for a faster download speed.

Tell them when they want to build it.

Comment (amilajack) [inline README.md]: I see. So do you want me to move this to the top of the `#Development` section?

File: README.md (hunk 3 above)
Comment (amilajack) [inline README.md]: Sorry, I'm not sure what you mean by this.

File: README.md (hunk 1 above)
Comment (Menci) [inline README.md]: You inserted `Development` section to the middle of `Building`. I think `Development` should be put after `Building`.

File: README.md (hunk 3 above)
Comment (Menci) [inline README.md]: There're three ways (not three steps) to open the devtools window: command line, config file, or shortcut key.

Comment (amilajack) [inline README.md]: I see. I'll add them.

**Pull request closed without merging**